### DISCUSSION TITLE
Modelling a thermocamera in FDS

### DISCUSSION THREAD

**Discussion by rkj15** [firemodels/fds#13074] [General]
Hello,

I was wondering if there is a way of modelling a thermal camera in FDS. I came across a case where a thermal camera is used to detect a fire. 
As the camera reads the infrared energy emitted from a fire source and converts that into apparent temperature I was thinking of using the a total heat flux device that would read activate when the specified flux is attained. However, I am not sure if this approach actually mimics the working of a thermal camera? The other question I have is how can I model the entire viewing angle of the camera? 

Or would there be another way of modelling this? Maybe using particles?

**Comment by drjfloyd** [comment]
An camera is more like a large collection of extremely narrow angle heat radiometers (not total heat flux as it doesn't measure convection). Each pixel is like its own heat flux gauge that only sees that pixel's worth of the overall camera view angle. If you were to greatly increase the number of radiation angles and set a correspondingly narrow VIEW_ANGLE for the radiometer you would get something representing a collection of pixels. You would have to work out what number of angles to set based on how the camera data was processed (I would asssume it required some fraction of its view to exceed a threshold and not just a single pixel).

**Reply by rkj15** [reply]
I had a feeling that my idea was wrong. I'll try to figure out the required radiation angles that gives us the expected result. Thank you for the response.
